src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -334,14 +339,18 @@ export class CalciteDropdown {
 
   private referenceEl: HTMLDivElement;
 
-  private dropdownFocusTimeout: number;
-
   //--------------------------------------------------------------------------
   //
   //  Private Methods
   //
   //--------------------------------------------------------------------------
 
+  transitionEnd = (event: TransitionEvent): void => {
+    if (event.propertyName === "opacity") {
```

Comment:
Does `transitionEnd` fire once for each transitioned property?  Is there a way to listen for when all of the applied transitions end so that you don't have to check specifically for `opacity` here?

Reply:
Yes it fires for each property, there's not a way to know when all are done.